todo-list/src/components/App/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+
+const list=[
+  {
+    task:"item1",
+    id:123
+  },
+  {
+    task:"item2",
+    id:12
+  },
+  {
+    task:"item3",
+    id:1
+  }
+];
+
+const App = () => {
+  return (
+  list.map((item)=> {
+    return (
+      <div>
```

Comment:
@Arsalan74 It still doesn't have `key` prop, even though you claimed you did so here: https://github.com/Arsalan74/To-Do-List/pull/2#issuecomment-623066932
Can you elaborate?